projects/fsastorefrontlib/src/cms-components/my-account/inbox/unread-messages-indicator/unread-messages-indicator.module.ts

```diff
@@ -0,0 +1,22 @@
+import { CommonModule } from '@angular/common';
```

Comment:
I think we don't need new module, you should add this new component to the existing inbox module.
Also please export unread-messages-indicator.component in the index.ts of inbox

Reply:
Ok, If I remove module I need to transfer this part `provideDefaultConfig(<CmsConfig>{
      cmsComponents: {
        UnreadMessagesIndicatorComponent: {
          component: UnreadMessagesIndicatorComponent,
        },
      },
    }),` into inbox.module in providers to ensure that component will be rendered in the view.

Reply:
After discussion, decided to go with separate module, just to import it on another place.